corehq/project_limits/shortcuts.py

```diff
@@ -11,3 +15,40 @@
 
 def get_standard_ratio_rate_definition(events_per_day):
     return STANDARD_RATIO.times(events_per_day)
+
+
+def delay_and_report_rate_limit(domain, max_wait, delay_rather_than_reject, datadog_metric, limiter):
```

Comment:
What is the rationale for putting this in the `shortcuts.py` file specifically? I may just not have a good understanding of that file's purpose in the first place.

Reply:
Oh, it just seemed like a fairly generic bit of code. Originally I reused it, but then I ended up deciding we specifically don't want delays repeaters (delays were the problem in the first place), but I liked pulling this out into something reusable so I kept it. It's technically not relevant to this PR anymore.

Reply:
As for `shortcuts.py`, I don't know, I just didn't know a better place to put it, and this seemed like it had some reusable utilities so I used it.

Reply:
Yeah my question wasn't about pulling it out into its own function, totally agree with that move. I just am unfamiliar with shortcuts.py files and if I were searching for rate limiting utilities, the shortcuts file is likely not where I'd look. All of this is moot since I'd just search for the function name anyway, but wanted to bring it up.

Reply:
Ah, yeah. I don't know if we have a consistent naming. I think of "shortcuts" as meaning things that are part of the API of the module, but are more nice-to-have additions than core functionality. We don't have a lot of files that use that convention, just a few